index.html

```diff
@@ -4242,39 +4572,57 @@ <h3>Parsing content</h3>
         <a>NFC content</a> which overwrites existing information.
         See also the [[[#writing-or-pushing-content]]] section.
       </p>
-      <p>
-        Since all local content that a web page has access to can be shared with
-        NFC, the user needs to be clearly aware about the permissions granted
-        to the web page using Web NFC.
-      </p>
-    </section>
-    <section> <h4>Store site URL as record identifier when writing data</h4>
-      <p>
-        When pushing an <a>NDEF message</a>, the [= host/registrable domain =],
-        <a data-cite="dom#concept-host-serializer">serialized</a>,
-        of the <a>current settings object</a> must be stored as the <a>record
-        identifier</a> in each top-level <a>NDEF Record</a>.
-      </p>
+      -->
     </section>
-    <section> <h4>Warn risk of physical location leak</h4>
+
+    <section> <h4>Warn about risk of physical location leak</h4>
       <p>
         When listening for and pushing <a>NFC content</a>,
-        the UA may warn the user that the given <a>origin</a> may be able to
+        the UA MAY warn the user that the given <a>origin</a> may be able to
         infer physical location.
       </p>
     </section>
+
     <section> <h4>Restrict automatic handling</h4>
       <p>
-        The payload data on <a>NFC content</a> is untrusted, and must not be used
-        by the UA to do automatic handling such as opening a web page
-        with a URL found in an <a>NFC tag</a>, unless the user approves that.
+        When the payload data on <a>NFC content</a> is untrusted, it MUST NOT
+        be used by the UA to do automatic handling of the content, such as
+        opening a web page with a URL found in an <a>NFC tag</a>, or installing
+        an application, or other actions, unless the user approves that.
       </p>
     </section>
-      <!--p>
-        For Bluetooth and WiFi handover (supported in later versions),
-        the user should have to grant access to the secondary API and must be
-        able to properly understand what they are granting.
-      </p-->
+
+    <section> <h4>Signing <a>NFC content</a></h4>
+      <div>
+        In this version of the specification the following policies are
+        recommended:
+        <ul>
+          <li>
+            A <a>smart poster</a> MAY be trusted only if signed by using
```

Comment:
Should we define explicitly that signature should cover whole NFC tag/poster in this case or only signed data should be considered trusted.

Reply:
The text says it should be signed with a single NDEF signature.
The question here is whether can we trust the smart poster if other records are also in the message. In that case we can trust only if it's the first record in the message or it's preceded by another signature record.